tests/devices/test_null_qubit.py

```diff
@@ -25,12 +24,18 @@
 from pennylane.devices.null_qubit import NullQubit
 from pennylane import Tracker
 
-from collections import defaultdict
 
+@pytest.fixture(
+    name="nullqubit_device",
+    scope="function",
+    params=[(np.float32, np.complex64), (np.float64, np.complex128)],
+)
+def fixture_nullqubit_device(request):
+    """Create a NullQubit device with the indicated number of wires,
+    parametrizing over real and complex dtypes."""
```

Comment:
Giving the fixture callable a different name and providing the name with which it is supposed to be used (via the `name` kwarg) avoids a Pylint message.